app/apps/client/utils/index.js

```diff
@@ -0,0 +1,16 @@
+function getRandomInt(max) {
+	return Math.floor(Math.random() * Math.floor(max));
+}
+
+/**
+ * Generate a random string with the specified length.
+ * @param length the length for the generated random string.
+ */
+export function randomString(length) {
```

Comment:
@lolimay meteor provides the `Meteor.random()` function, it may be possible to replace these functions with it

Reply:
Resolved in #15781